### ISSUE TITLE
About the Transferability of your method

### ISSUE THREAD

**Issue by Teiboku** [annegnx/PnP-Flow#24]
Hi, Dear authors,
I am very interested in applying this method to real-world image denoising, based on the descriptions and training strategy presented in the paper. I would greatly appreciate your insights on the following questions:

For more complex data distributions (e.g., low-light images, HDR images), would FlowMatching require significantly larger training datasets compared to the current setup in order to achieve good performance?
When dealing with more complex real-world degradations (such as ISP-processed noise, motion blur, etc.), can the robustness of this method still be maintained?

Thank you very much for your time and help!

**Comment by SegoleneMartin** [comment]
Hi, thanks for reaching out! I believe it would be very interesting to try the PnP-Flow method on some harder datasets as you suggest. We did not do it ourselves and the method might require some tuning (the gradient step parameter for instance).

I would expect that for complex datasets (I don't know how "complex" low-light images are in terms of distribution -- for me a complex data set is also a diverse data set) you need more data and to train longer. This being said, it's difficult to put numbers on how many images you should have in your training. You could take Cifar10 as a reference for instance (which is a not so easy image dataset): it has 5000 training samples per class (10 classes), and one usually trains a FM for at least 500 epochs with batch size 128. The first step is to evaluate if your generated samples look reasonable before trying PnP-Flow.

Don't hesitate to keep us updated!
